packaging/run_inference.py

```diff
@@ -0,0 +1,228 @@
+import os
+import argparse
+import torch
+from pathlib import Path
+from batchgenerators.utilities.file_and_folder_operations import join
+import time
+
+from nnunetv2.inference.predict_from_raw_data import predict_from_raw_data as predictor
+# from nnunetv2.inference.predict_from_raw_data import nnUNetPredictor
+
+
+"""
+Usage example:
+Method 1 (when running on whole dataset):
+    python run_inference.py --path-dataset /path/to/test-dataset --path-out /path/to/output --path-model /path/to/model
+
+Method 2 (when running on individual images):
+    python run_inference.py --path-images /path/to/image1 /path/to/image2 --path-out /path/to/output --path-model /path/to/model
+"""
+
+def get_parser():
+    # parse command line arguments
+    parser = argparse.ArgumentParser(description='Segment images using nnUNet')
+    parser.add_argument('--path-dataset', default=None, type=str,
+                        help='Path to the test dataset folder. Use this argument only if you want '
```

Comment:
Maybe we could explicitly mention the expected dataset structure (probably nnUNet structure, i.e., not BIDS)

Reply:
That is not required! See [these lines](https://github.com/ivadomed/model_seg_sci/blob/c302c816d31275b24d87ce0b2b776bc714d7f6cf/packaging/run_inference.py#L117-L126)! :)

Reply:
Okay, cool! Sorry, I missed the lines.

Reply:
Okay! Filenames can be BIDS! But not the whole dataset 😄